pkg/customMetrics/customMetrics.go

```diff
@@ -0,0 +1,30 @@
+package customMetrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+var (
+	RegistryCallsTotal = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "registry_calls_total",
```

Comment:
@StevenTobin lets add a prefix to these. e.g. `heimdall_` so that they are more searchable in prometheus.